osu.Game/Screens/SelectV2/WedgeHitStatisticsDisplay.cs

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using System.Linq;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Layout;
+using osu.Game.Beatmaps;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Overlays;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Screens.SelectV2
+{
+    public partial class WedgeHitStatisticsDisplay : CompositeDrawable, IHasAccentColour
+    {
+        private const float statistic_width = 75;
```

Comment:
Can probably reduce this since `Count` is gone (at least to 65 to match the right).

Reply:
`Banana Showers` in osu!catch still requires this set to 75.

Reply:
Roughly improved in https://github.com/ppy/osu/commit/6ca36855e46459d0f06a29fc144465eb0d883f41 as well.